src-bindings/polka/polka.ml

```diff
@@ -0,0 +1,69 @@
+open Interop
+
+module Server = struct
+  include Interface.Make ()
+
+  module Address = struct
+    include Interface.Make ()
+
+    include [%js: val port : t -> int [@@js.get]]
+  end
+
+  include
+    [%js:
+    val close : t -> t [@@js.call]
+
+    val address : t -> Address.t [@@js.call]
+
+    val on : t -> string -> Ojs.t -> unit [@@js.call]]
+
+  let on t = function
+    | `Close f -> on t "close" @@ [%js.of: unit -> unit] f
+    | `Error f -> on t "error" @@ [%js.of: err:Node.JsError.t -> unit] f
+end
+
+type polka = Ojs.t
```

Comment:
Any idea how I could use `type t` instead? The reason why I had to change it to `polka` is to not conflict with the `Middleware.t`.

On the `mli` file I could do:

```ocaml
module Middleware : sig
  type polka := t
  type t
end
```

But here I don't know how to do that 😅

Reply:
hmm, `type nonrec t = ...` should work? I'd say type `polka` is more readable when used in `Middleware.t`'s definition

Reply:
Let's stick with `polka` then :P